### ISSUE TITLE
Satellite tutorials feedback form questions:

### ISSUE THREAD

**Issue by sanjayankur31** [OCNS/SoftwareWG#109]
@OCNS/software-wg we wanted a short feedback form to gather some information from registrants on sessions they attended, didn't attend, and what we can do to increase attendance. 

How are these:

- What sessions did you attend?
- Do you have any general feedback for tutors for the sessions you attended?
- What sessions did you plan to attend but did not, why?
- What can we do to make it easier for you to attend more sessions?
- Any other general comments.

Thoughts? It'll be good to have these fleshed so I can send the e-mail out to them on Thursday (14th July)

**Comment by appukuttan-shailesh** [comment]
Along the lines of what we discussed during the meeting:
- Would you be open to paying a small registration fee to attend such tutorials, if they are accompanied with an official certificate of attendance ?
- Do you have any suggestions for making these online events more interactive and engaging?
- Would you be interested in a similar week of online tutorials sometime around the start of next year (e.g. early February)?

**Comment by appukuttan-shailesh** [comment]
As part of a survey/feedback I would still be interested in seeing how people respond to the token payment for such tutorials (maybe just out curiosity :p).

But, yes, the intended purpose is to have a much better idea of the eventual turnout. As we discussed, for free online events it is normal that the turnout is always much lower. But when it goes down to ~15% we have a problem. It is imperative that we improve this situation as quite a few presenters expressed their surprise (and evident disappointment).

**Comment by sanjayankur31** [comment]
> As part of a survey/feedback I would still be interested in seeing how people respond to the token payment for such tutorials (maybe just out curiosity :p) even if we don't actually enforce it.

I am curious too, but I am hesitant to ask "would you register if we charged a token fee"? Given that without the token fee we had few final attendees, my expectation is that maybe a subset of the attendees would perhaps say "yes", but everyone else will just say "no".  If we had 100s of attendees which indicated that the sessions we well attended, asking if folks would still attend would get us some info, but at the moment, I don't see why folks would say "yes" if they didn't actually attend the session. The question also adds other complications. It'll make people wonder:

- what is this "token amount": depending on what it is, folks will worry that they may not get reimbursed etc. and even if it is "low" for developed societies, it may not be low for others.
- what is the money for, given that everything we're using is free

Basically, i don't think it'll give us any useful information, and the thought required to frame the question to be able to get any useful information is not worth it. So I'd rather focus on "what can we do to improve turnout" than "what can we do to have more accurate registration numbers".

**Comment by appukuttan-shailesh** [comment]
As proposed during the meeting, any fee based registration would likely additionally offer them additional incentives e.g. participation certificate (thereby differentiating it from our current free setup).

We could maybe consider offering these certificates even with our free setup, whereby candidates who actually attend the session (and maybe additionally complete an assignment/exercise as "proof" of engagement). Could be an incentive for better attendance.

**Comment by sanjayankur31** [comment]
> As proposed during the meeting, any fee based registration would likely additionally offer them additional incentives e.g. participation certificate (thereby differentiating it from our current free setup).

Yes, but we don't currently know what these are (and how much extra work this is for us as organisers in terms of logistics etc., and for tutors in terms of preparation, delivery, and post-tutorial work too), so I'd rather not include them in the current feedback form. It's something we can discuss with tutors instead.

In general, while we discussed charging, our discussion leaned more towards not charging because of all the extra work + complexity it would entail without any real benefits in terms of attendance.

> 
> We could maybe consider offering these digital certificates even with our free setup, whereby candidates who actually attend the session (and maybe additionally complete an assignment/exercise as "proof" of engagement) are considered eligible. Could be an incentive for better attendance.

Yeh, that could be done, but it has implementation issues that need figuring out---what counts as "proof of engagement" for a start? 

We can discuss all of these in the future if/when we decide to do another round of tutorials. In the meantime, let's skip charging related queries from the form in this iteration since our focus here is on improving attendance, not reducing registration?

**Comment by appukuttan-shailesh** [comment]
I am quite open to keeping the feedback form more exploratory to (try to) better understand our audience (even though their responses are never binding.... as we saw with the registration forms this time).

But we can keep it simple this time, and go with the basic questions.

**Comment by brenthuisman** [comment]
Do we know anything about the signups? Can we say that signups from $institute_X or $country_Y have a habit of being no-shows? Or maybe people without any affiliation? Or students (is that a question on the signup form?)? Even if we change nothing, if there's a pattern like that, we could adjust the expected number of attendees for our own reference.

**Comment by sanjayankur31** [comment]
Hrm, not really. We have their e-mail addresses, but generally, it's quite a diverse set of folks. We didn't explicitly ask what folks were doing or where they were based etc. because we wanted the form to be as short as possible (and we didn't think we'd need the extra info).

**Comment by sanjayankur31** [comment]
Here's the feedback form folks. Could you all please take a look and let me know what you think? I can then send it out to registrants next week:

https://framaforms.org/cns2022-online-satellite-tutorials-feedback-form-1659102112
